src/VkBootstrap.h

```diff
@@ -440,6 +440,7 @@ struct PhysicalDevice {
 	operator VkPhysicalDevice() const;
 
 	private:
+	char device_name[VK_MAX_PHYSICAL_DEVICE_NAME_SIZE];
```

Comment:
Possibly left in by accident? Since the name is in `VkPhysicalDeviceProperties` and this is a private variable.